mtp_api/apps/prison/serializers.py

```diff
@@ -80,6 +87,34 @@ class Meta:
         )
 
 
+class PrisonerAccountBalanceSerializer(serializers.Serializer):
+    combined_account_balance = serializers.SerializerMethodField()
+
+    def get_combined_account_balance(self, prisoner_location: PrisonerLocation):
+        if prisoner_location.prison.private_estate:
+            # NB: balances are not known in private estate currently
+            return 0
+
+        try:
+            nomis_account_balances = nomis.get_account_balances(
+                prisoner_location.prison.nomis_id,
+                prisoner_location.prisoner_number,
+            )
+            return (
+                nomis_account_balances['cash'] +
+                nomis_account_balances['spends'] +
+                nomis_account_balances['savings']
```

Comment:
Will Nomis values are always an integer, for instance for a new prisoner with no activity in savings for instance will definitely be 0?

Reply:
the docs aren't explicit, but i've found nothing to suggest otherwise. at least we will immediately get an alert if an account is missing or has an unexpected type

Reply:
Does the documentation specify anything about the types of these fields at all? If not, then can we add an explicit assertion on these types?

Reply:
they say "All balance values are represented as pence values" with a sample response of `{  "cash": 13565,  "savings": 12344,  "spends": 5678}`. i'm happy to add some assertions